main.go

```diff
@@ -2,15 +2,15 @@ package main
 
 import (
 	"github.com/gin-gonic/gin"
-	"se-school-case/controller"
-	"se-school-case/initializer"
-	"se-school-case/service"
+	controller2 "se-school-case/pkg/controller"
+	initializer2 "se-school-case/pkg/initializer"
+	"se-school-case/pkg/service"
 )
 
 func init() {
-	initializer.LoadEnvVariables()
-	initializer.ConnectToDatabase()
-	initializer.RunMigrations()
+	initializer2.LoadEnvVariables()
+	initializer2.ConnectToDatabase()
```

Comment:
Can we remove controller2, initializer2 aliases, and just use controller and initializer ?